cli/cmd/root.go

```diff
@@ -4,13 +4,15 @@ import (
 	"context"
 	"os"
 
+	"github.com/rilldata/rill/cli/cmd/docs"
 	"github.com/rilldata/rill/cli/cmd/dropsource"
 	"github.com/rilldata/rill/cli/cmd/example"
 	"github.com/rilldata/rill/cli/cmd/importsource"
 	"github.com/rilldata/rill/cli/cmd/info"
 	"github.com/rilldata/rill/cli/cmd/initialize"
 	"github.com/rilldata/rill/cli/cmd/start"
 	"github.com/rilldata/rill/cli/cmd/version"
+
 	"github.com/spf13/cobra"
```

Comment:
We've begun listing all non-standard dependencies in one group (based on Uber's style guide on [import group ordering](https://github.com/uber-go/guide/blob/master/style.md#import-group-ordering)), so just remove the newline here